deploy/11_fundSpc.ts

```diff
@@ -13,6 +13,8 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const spc = await ethers.getContract('Spc') as Spc;
   console.log(`Funding the SPC at ${spc.address} with 10_000 ETH...`);
   spc.connect(storageSigner).provisionWithEth({ value: toBaseUnit(10_000, 18) });
+
+  await wait(8000)
```

Comment:
Why do we `wait` ? :)

Reply:
Because if we queue all the transactions the gas fee increase and the deployments fail in the middle.

Reply:
Ah ok!